src/attributes/components/AttributeOrganization/AttributeOrganization.tsx

```diff
@@ -73,12 +75,12 @@ const AttributeOrganization: React.FC<AttributeOrganizationProps> = props => {
             label={
               <>
                 <FormattedMessage id="v1pNHW" defaultMessage="Attribute Class" />
-                <Typography variant="caption">
+                <Text fontWeight="light">
```

Comment:
question: `variant="caption"` has different font color, is there any reason not to apply it here?